build.assets/Dockerfile

```diff
@@ -188,6 +207,21 @@ RUN curl --proto '=https' --tlsv1.2 -sSf https://sh.rustup.rs | sh -s -- -y --pr
     if [ "$BUILDARCH" = "amd64" ]; then rustup target add i686-unknown-linux-gnu arm-unknown-linux-gnueabihf aarch64-unknown-linux-gnu; fi && \
     cargo install cbindgen
 
+# Do a quick switch back to root and copy/setup libfido2 binaries.
+# Do this last to take better advantage of the multi-stage build.
 USER root
+COPY --from=libfido2 /usr/local/include/ /usr/local/include/
+COPY --from=libfido2 \
+    /usr/local/lib/libcbor.a \
+    /usr/local/lib/libfido2.a \
+    /usr/local/lib/libfido2.so.1.10.0 \
```

Comment:
I would avoid building/copying the shared library altogether, to be sure we don't accidentally link it. (same applies to the centos build)

Reply:
Wouldn't that fail to test problems that might occur as a result of using the statically linked library?

Reply:
Testing mocks fido2 interactions, so unfortunately no. We do need the libraries to `go get` and compile go-libfido2.

It would be interesting to exercise and `ldd` our release binaries, to make sure we have no regressions, but that must happen outside of the scope of the Dockerfile (they don't actually capture the sources).